### DISCUSSION TITLE
How do I configure the control target Set Value (SV) using event buttons

### DISCUSSION THREAD

**Discussion by Terracotta-62** [artisan-roaster-scope/artisan#1114] [Q&A]
![20230227214910](https://user-images.githubusercontent.com/116416332/221581125-90b20219-bd27-4e49-9e27-651186df3108.png)

**Comment by MAKOMO** [comment]
You can find this information in the [documentation of event buttons](https://htmlpreview.github.io/?https://github.com/artisan-roaster-scope/artisan/blob/master/doc/help_dialogs/Output_html/eventbuttons_help.html). There are two `Artisan Command`s to modify the SV which you can assign to an event button of your choice. `adjustSV(<int>)`and `pidSV(<int>)`

**Comment by Terracotta-62** [comment]
Thanks for your help. It's done.
